internal/driver/driver.go

```diff
@@ -94,32 +96,52 @@ func (d *Driver) Initialize(sdk interfaces.DeviceServiceSDK) error {
 		return fmt.Errorf("failed to add API route %s, error: %s", ApiRefreshDevicePaths, err.Error())
 	}
 
-	rtspServerHostName, ok := d.ds.DriverConfigs()[RtspServerHostName]
-	if !ok {
-		rtspServerHostName = DefaultRtspServerHostName
-		d.lc.Warnf("service config %s not found. Use the default value: %s", RtspServerHostName, DefaultRtspServerHostName)
+	var err error
+	d.enableRtspServer, err = strconv.ParseBool(d.ds.DriverConfigs()[EnableRtspServer])
+	if err != nil {
+		return fmt.Errorf("failed to parse EnableRtspServer config: %s", err.Error())
 	}
-	d.lc.Infof("RTSP server hostname: %s", rtspServerHostName)
-	d.rtspHostName = rtspServerHostName
+	if d.enableRtspServer {
+		// check to see if rtsp-simple-server file/binary exists
+		_, err := os.Stat(RtspServerCmd)
+		if err != nil {
+			if os.IsNotExist(err) {
+				return fmt.Errorf("%s file cannot be found: %s", RtspServerCmd, err.Error())
+			}
+		}
+		rtspProc := exec.Command(RtspServerCmd)
+		rtspProc.Stdout = os.Stdout
+		err = rtspProc.Start()
+		if err != nil {
+			return fmt.Errorf("unable to start %s process: %s", RtspServerCmd, err.Error())
+		}
```

Comment:
I think I would move the starting of the process to the end of this if block instead of the beginning